members/SARL35_283916.yaml

```diff
@@ -0,0 +1,54 @@
+bioguide: SARL35_283916
+contact_form:
+  method: POST
```

Comment:
lets go ahead and start leaving this out of new yamls since it is unused

Reply:
deleted